samples/ControlCatalog/MainWindow.xaml.cs

```diff
@@ -24,5 +39,12 @@ private void InitializeComponent()
             theme.TryGetResource("Button", out _);
             AvaloniaXamlLoader.Load(this);
         }
+
+        protected override void OnTemplateApplied(TemplateAppliedEventArgs e)
+        {
+            base.OnTemplateApplied(e);
+
+            _notificationArea.Install(this);
```

Comment:
Having to do this feels a little messy. Could the `WindowNotificationManager` take the `Window` in its ctor, and if the window has no visual children listen for the template being applied?

Reply:
fixed :)